build/images/ovs/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM ubuntu:20.04 as ovs-debs
 
 # Some patches may not apply cleanly if another version is provided.
-ARG OVS_VERSION=2.13.1
+ARG OVS_VERSION=2.14.0
```

Comment:
they are several more places in the Antrea repo that need to be updated. Grep for `2.13.1`.

Reply:
Thanks for reminding. Updated.
Please note that I also updated the OVS for Windows. It has no been tested and uploaded yet. Will merge this PR after that Windows OVS v2.14.0 is ready.